pennylane/templates/subroutines/double_excitation_unitary.py

```diff
@@ -471,13 +474,12 @@ def DoubleExcitationUnitary(weight, wires1=None, wires2=None):
             dev = qml.device('default.qubit', wires=5)
 
             @qml.qnode(dev)
-            def circuit(weight, pphh=None):
```

Comment:
Was this example previously out of date with the docstring?

Reply:
Yes, when we did the generalization of the templates we missed this :-(.